### PR TITLE
Tandem export/count fixes

### PR DESCRIPTION
**Pull request by mjoyce91**
- Fixes the count/tandem_count functions were swapped
- Passes `isTandem=True` to the mapping function for the tandem export

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
@mjoyce91 might need to also be implemented for Projected Vacancies?

**Comment by elizabeth-jimenez** [comment]
**Test Case**
Tandem Filters: Danger Pay: 20%
Tandem1 Filters: **Skill** -> General Services -> Transportation and Freight (2335)
**Grade** -> OC Counselor (FE-OC)
Tandem2 Filters: **Skill** -> Consular -> CONSULAR AFFAIRS (3001)

On Open "Viewing 1-5 of 21 Results" Exports 21 Results
On PV "Viewing 1-5 of 9 Results" Exports 28 Results

**Comment by mjoyce91** [comment]
thanks @elizabeth-jimenez I'll take a look

File: talentmap_api/fsbid/services/available_positions.py [hunk 1]
```diff
@@ -131,7 +131,7 @@ def get_available_positions_tandem_csv(query, jwt_token, host=None, limit=None,
     data = services.send_get_csv_request(
         "positions/available/tandem",
         query,
-        convert_ap_query,
+        partial(convert_ap_query, isTandem=True),
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/available_positions.py:134]: Also doing this in `talentmap_api/fsbid/services/projected_vacancies.py` for `get_projected_vacancies_tandem_csv`seems to fix it for projected :)

Comment (mjoyce91) [reply talentmap_api/fsbid/services/available_positions.py:134]: Fixed

**Pull request merged**